core/templates/dev/head/base.js

```diff
@@ -95,6 +97,11 @@ oppia.controller('Base', [
 
     // If this is nonempty, the whole page goes into 'Loading...' mode.
     $rootScope.loadingMessage = '';
+    // Method to call windowSize on resizing the window
+    $window.onresize = function() {
```

Comment:
@giritheja -- I was looking at a commit by @disissid and realized that this might overwrite any existing behavior declared on $window.onresize(). Is this the case? If so, it might be better to find a non-overwriting method of doing this, or alternatively register onresize hooks to windowDimensionsService to be called one after the other on a resize event.